src/main/java/org/terasology/structureTemplates/internal/systems/StructureSpawnServerSystem.java

```diff
@@ -103,6 +108,8 @@
 
     private BlockRegionTransform regionTransform;
     private EntityRef structureEntity;
+    Vector3ic location = new Vector3i();
+    List<SpawnBlockRegionsComponent.RegionToFill> blockRegionList;
```

Comment:
I think these top-level members should have some Javadoc explaining what they are used for. I would not expect that the ST server system needs a location or list of block regions on its own.